frontend/ember-cli-build.js

```diff
@@ -9,11 +9,15 @@ module.exports = function(defaults) {
   });
   app.import('vendor/js/jquery.js')
   app.import('vendor/js/jquery.tmpl.js')
-  app.import('vendor/js/bootstrap.min.js')
+
+  app.import('bower_components/jquery/dist/jquery.min.js');
```

Comment:
```hbs
 
{{#each model as |item index|}}
	<li>
		{{item.filename}}
		{{item.Date}}
		{{item.Id}}
	</li>
{{else}}
	Sorry, no scripts
{{/each}}
```

Iteration should work if your model output is 

```
[{"filename":"somename","Date":"Date string","Id":1}]
``` 

as an object. 

What're you're getting is a string instead. You should be using `Ember.$.getJSON` instead of `Ember.$.get`